airflow/www/static/js/dags.js

```diff
@@ -61,9 +61,10 @@ $("#tags_filter").select2({
 $("#tags_filter").on("change", (e) => {
   e.preventDefault();
   const query = new URLSearchParams(window.location.search);
-  if (e.val.length) {
```

Comment:
This issue is because those tags are not actually on the page, so all we need to do is update this check, right?

`if (e.val && e.val.length) {`